packages/dapp/src/const/trackingKeys.ts

```diff
@@ -0,0 +1,29 @@
+export enum trackingCategories {
+  language = 'language',
+  menu = 'menu',
+  navigation = 'navigation',
+  supportModal = 'support-modal',
+  theme = 'theme-menu',
+  themeSwitch = 'theme-switch',
+  wallet = 'wallet',
+  widgetEvent = 'widget-event',
+}
+
+export enum trackingActions {
+  addChain = 'add-chain',
+  addToken = 'add-token',
+  clickThemeSwitch = 'click-theme-switch',
+  chooseWallet = 'choose-wallet',
+  disconnect = 'disconnect',
+  onRouteExecutionStarted = 'onRouteExecutionStarted',
+  onRouteExecutionCompleted = 'onRouteExecutionCompleted',
+  onRouteExecutionFailed = 'onRouteExecutionFailed',
+  onRouteExecutionUpdated = 'onRouteExecutionUpdated',
+  openConnectedMenu = 'open-connected-menu',
+  openSubmenu = 'open-submenu',
+  openSupportModal = 'open-support-modal',
+  switchChain = 'switch-chain',
+  switchLanguage = 'switch-language',
+  switchTab = 'switch-tab',
+  switchTheme = 'switch-theme-mode',
+}
```

Comment:
Let's stick to the [classic](https://www.typescriptlang.org/docs/handbook/enums.html) naming convention of enums and their properties here 🙂

Reply:
@chybisov I hope that I hit the nail with those adjustments in my last commit? ⚒️

Reply:
@dennyscode sorry, but where did you find UPPER_CASE there? 😅

I meant this

```ts
enum Direction {
  Up,
  Down,
  Left,
  Right,
}
```


I should be more specific next time 😅